### DISCUSSION TITLE
Submarine discussion and comparing

### DISCUSSION THREAD

**Discussion by Therightroad2009** [SoftbearStudios/mk48#210] [General]
**Interesing Comparison between Yasen and Seawolf**
-
Here i did a comparison between Yasen and Seawolf in a image.  (Mostly focused in Seawolf) and you can see that according to FIREPOWER stats. 
Seawolf can do 9.86 damage to both ships and submarines if their 8 MK-48s hit.

Yasen can do 10.26 damage to ships or surfaced submarines using 4 missiles and 6 torpedos (EXCLUDING THE VODOPAD)
Since Missiles are useless for submarine fights, Yasen would use only 6 torpedos and it would do 6.9 damage
 **(2.96 less than seawolf can do)** If we did 6 torpedos + 4 Vodopads it would be 9.38 **which is still a bit less than Seawolf.** 

This is not all...

Yasen has 4 decoys (Brosok decoys are better than MOSS ones) and more health than Seawolf. But it's bigger and easier to hit/detect.
Meanwhile Seawolf has only 2 MOSS, It's stealth and has better sensors 

**Akula Sensors**
-
I'd like to say this experience i had which made me have doubts:

Once i was an Akula with active sensors off going to south.
I saw a passive sonar contact at south that was heading to west at like 5 knots. When i got very close, it was a Seawolf || and guess what? It was very close that i almost collide with it. 

**Does this apply for the other submarines because Akula sonar is bad?**

**Is it because of Seawolf Stealth?** 

**MOSS Issues??**
-
Once i was a Ohio heading to Northeast and i saw a torpedo coming from Southwest and heading to my position, **(ACTIVE SENSORS WERE OFF)**

I threw a MOSS to Southeast and the torpedo didn't chase it.
The distance between the torpedo and me was middle, so i wonder why didn't the torpedo chase the MOSS instead.

**A small suggestion**
-
Make torpedos ping if you are inside their seeker like real life.
"""""
This is all of it, you can use this post to discuss about submarines :)

**Comment by finnbear** [comment]
Lots of good anecdotes here :heart: 

>  i wonder why didn't the torpedo chase the MOSS instead.

Torpedo homing is based on many factors but, even if they were all in favor of the decoy, there is a small bit of randomness associated with each torpedo/target pair to discourage entire salvos of torpedoes from locking on to a the same target.

> Make torpedos ping if you are inside their seeker like real life.

The current client/server network synchronization architecture does not permit this, but I'm working on a complete rewrite of that part of the code that will make it easy. It will probably also be able to tell you if/when an enemy ship detects you :)

**Reply by Therightroad2009** [reply]
Thanks for the quick reply, i wonder if the tier 10 submarine is already choosen and when it will be on progress?

> It will probably also be able to tell you if/when an enemy ship detects you :)

This is awesome, i hope it happens.

**Comment by Therightroad2009** [comment]
@finnbear I was testing submarine sensors with a friend and i made this list (read it carefully please)


All of these tests were made at 0 knots.

METHOD USED FOR TESTS: **nashe** helped me for making these tests with the next method:
 I picked Akula and he picked the sub we wanted to test sensors in.
 Every submarine we wanted to test put to detect an Akula, so submarines with stealth will slighty reduce the values seen.
 So basically i moved to the exact point when he detected me, and in that exact coordinate. I asked him his coordinates and i substracted his coordinates by my coordinates.


SKIPJACK SENSOR

SONAR

Skipjack Active Sonar Range: 594 (WHITE LINE VISIBLE)
Skipjack Passive Sonar Range: 258 (WHITE LINE VISIBLE)

AKULA SENSOR

SONAR

Akula Active Sonar Range: 640 (WHITE LINE VISIBLE)
Akula Passive Sonar Range: 286 (WHITE LINE VISIBLE)

OHIO SENSOR

SONAR

Ohio Active Sonar Range: 710 (WHITE LINE VISIBLE)
Ohio Passive Sonar Range: 308 (WHITE LINE VISIBLE)

YASEN SENSOR

SONAR

Yasen Active Sensor Range: 640 (WHITE LINE VISIBLE)
Yasen Passive Sensor Range: 286 (WHITE LINE VISIBLE)

SEAWOLF SENSOR

SONAR

Seawolf Active Sonar Range: 624 (WHITE LINE VISIBLE)
Seawolf Passive Sonar Range: 280 (WHITE LINE VISIBLE)


Basically this is pretty unrealistic because we can see how Ohio has better range than all of these, Akula and Yasen have same sonar, and Seawolf has worse or almost the same sonar than Yasen or Akula, the order in the game from worst to best is most unrealistic thing i've ever seen:


from worst to best
**ACTUAL GAME**
1. Skipjack
2. Seawolf
3. Akula and Yasen
4. Ohio


**HOW IT SHOULD BE**
1. Skipjack
2. Akula
3. Ohio
4. Yasen
5. Seawolf 

Idk what does the game use for defining range, but if it uses the method i use i'd recommend making this:

SKIPJACK active sonar from 594 to 520 and passive sonar from 258 to 220.

AKULA active sonar from 640 to 600 and passive sonar from 288 to 240.

OHIO active sonar from 710 to 630 and passive sonar from 308 to 275.

YASEN active sonar from 640 to 650 and passive sonar from 288 to 300.

SEAWOLF active sonar from 624 to 710 and passive sonar from 280 to 310.


It's a simple and realistic update ngl. This should be done asap

**Reply by finnbear** [reply]
Can you provide some citations for why your ranking is more realistic? I'd like a primary/secondary source.

> Idk what does the game use for defining range

Right now, the game assumes that larger submarine means better sonar.

**Reply by Therightroad2009** [reply]
@finnbear 

> Can you provide some citations for why your ranking is more realistic? I'd like a primary/secondary source.

Skipjack is a 60ish sub that uses old sonars you, it's a very old sub. Means it's sonars will never be good as current ones. 

Akula is also a old cylindrical bow mounted sonar, i think it shoudln't be better as Seawolfs spherical modern sonar

Ohio is a old gen BQQ-6 with those fat screens old cold war look. 

Yasen can be same as seawolf but the noise it makes means sonar will be less sensitive, and the less sensitive sonar, the smaller range. 

And Seawolf is still on top with virginia,there is high chance Yasen can be good as Seawolf but that isn't realistic because Russia uses old tech that US is not going to use anymore. Russia it's like 30 years behind lol. I've saw video footage from Seawolf sonar screen from close before, dude they are very sensitive.

**Comment by skydengyao** [comment]
您好！  您发的邮件已收到，谢谢！
